apis/docdb/v1alpha1/zz_db_subnet_group.go

```diff
@@ -0,0 +1,109 @@
+/*
+Copyright 2021 The Crossplane Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Code generated by ack-generate. DO NOT EDIT.
+
+package v1alpha1
+
+import (
+	xpv1 "github.com/crossplane/crossplane-runtime/apis/common/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+)
+
+// DBSubnetGroupParameters defines the desired state of DBSubnetGroup
+type DBSubnetGroupParameters struct {
+	// Region is which region the DBSubnetGroup will be created.
+	// +kubebuilder:validation:Required
+	Region string `json:"region"`
+	// The description for the subnet group.
+	// +kubebuilder:validation:Required
+	DBSubnetGroupDescription *string `json:"dbSubnetGroupDescription"`
+	// The name for the subnet group. This value is stored as a lowercase string.
+	//
+	// Constraints: Must contain no more than 255 letters, numbers, periods, underscores,
+	// spaces, or hyphens. Must not be default.
+	//
+	// Example: mySubnetgroup
+	// +kubebuilder:validation:Required
+	DBSubnetGroupName *string `json:"dbSubnetGroupName"`
```

Comment:
We're able to use external-name annotation as name, so we need to [ignore](https://github.com/crossplane/provider-aws/blob/master/CODE_GENERATION.md#external-name) this field and rely on that.

Reply:
Fixed